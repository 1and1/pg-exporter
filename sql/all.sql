select current_setting('server_version_num')::int / 10000 as pg_major_version;

-- remove all values from the stats
-- remove database ids
-- remove Go and PostgreSQL version numbers
-- remove postgresql_locks_count{mode="RowExclusiveLock"} line which sometimes pops up
-- remove debug_discard_caches help since its max value depends on --enable-cassert
\! curl -sf localhost:9135/metrics | sed -e 's/^\([a-z].*\) .*/\1/' -e 's/database_id="[0-9]*"/database_id=""/' -e 's/version="[^"]*"/version=""/' -e '/mode="RowExclusiveLock"/d' -e '/# HELP postgresql_settings_debug_discard_caches/d'
