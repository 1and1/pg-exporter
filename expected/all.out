This file is intentionally left empty, see all_*.out instead.
